### PR TITLE
Update second-life-viewer from 6.4.23.564172 to 6.5.0.565607

### PR DESCRIPTION
**Pull request by neutric**
Created with `brew bump-cask-pr`.

### CODE REVIEW

File: Casks/second-life-viewer.rb [hunk 1]
```diff
@@ -11,6 +11,8 @@
     url "https://secondlife.com/support/downloads/"
     strategy :page_match do |page|
       v = page[%r{href=.*?/Second_Life_(\d+(?:_\d+)*)_x86_64\.dmg}i, 1]
+      next if v.blank?
+
```
Comment (neutric) [inline Casks/second-life-viewer.rb:15]: @bevanjkay The Safe Navigation Operator `&` in `v&.tr("_", ".")`, which I was about to add, should have the same effect. Any preference?

Comment (bevanjkay) [reply Casks/second-life-viewer.rb:15]: @neutric I don't have a preference - `next if v.blank?` is more consistent with the other applications.

Comment (neutric) [reply Casks/second-life-viewer.rb:15]: Let's go with yours then. Can you please replace the second `*` quantifier with a `+` as well?

Comment (bevanjkay) [reply Casks/second-life-viewer.rb:15]: Thanks, missed that one 😊

**Pull request merged**